test/resources/roles/testplugin/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2017 John Dewey
```

Comment:
RH?

Reply:
This code is copy/pasta'd from https://github.com/retr0h/ansible-plugins to avoid network acess.

My guess is we need to retain the license.